instructions/section-1/lesson-1-1.md

```diff
@@ -0,0 +1,129 @@
+[< Back to Overview](../../README.md)
+
+# Lesson 1.1: Introduction
+
+This assignment will teach you the following:
+
+- Hello React
+- Project Setup
+- React components
+- JSX in React
+- Lists in React
+
+---
+
+## Instructions
+
+### Getting Started:
+
+Fork this GitHub repository to create your own working version:
+
+https://github.com/Code-the-Dream-School/ctd-react-egret
+
+After completing the steps, you should be directed to your forked GitHub repository:
+
+![Fork Assignment](../assets/section-1/lesson-1-1-fork.png)
+
+Click the "Code" button and copy the URL for your repository:
+
+![Clone Assignment](../assets/section-1/lesson-1-1-clone.png)
+
+Clone this repository so that the code is available on your local machine:
+
+> Hint: Run this command from the directory where you want to store your code
+
+> Note: Replace `<url>` with the URL you copied in the previous step
+
+    git clone <url>
+
+Navigate to the directory you just cloned in your terminal:
+
+    cd ctd-react-egret
+
+Create a new local branch to work on separate from the `main` branch:
+
+    git checkout -b lesson-1-1
+
+Now, open the project directory in your code editor and continue to the next section.
+
+### Task List:
+
+#### Project Setup
+
+In your terminal, use the `create-react-app` script to generate a new React project:
+
+> Note: the `.` specifies that the app should be created in the same directory and the `--template` specifies what project template to use
+
+    npx create-react-app . --template minimal
```

Comment:
@hifilorau, @siirila opted for this "minimal" template instead of the default one so reduce fluff and keep students focused on the bare bones of a React app - thoughts?